test/unit_tests/chess_compiler_tests/README.md

```diff
@@ -0,0 +1,13 @@
+There are two ways to use precompiled functions in your code.
+
+## 1) Precompiled kernel (tests 02 and 04)
+
+In this method, the *kernel.cc* manages the locks and is compiled to an elf file, which can be used to program the AIE.
+
+You can modify and recompile the *kernel.cc* without recompiling the host.
+
+## 2) Precompiled core function (tests 01, 03, 05, and 07)
+
+The *kernel.cc* does not manage the locks in this method and is compiled to an objective file.
```

Comment:
What *does* manage the locks?

Reply:
Locks are managed in aie.mlir. ReadMe is adjusted to clarify this.